schema/common.mod

```diff
@@ -83,10 +83,18 @@
 	in multi-staff music. For example, the stopping point for
 	a slur may appear in staff 1 before the starting point for
 	the slur appears in staff 2 later in the document.
+
+	The start-stop-continue-let-ring type extends the

```

Comment:
I think we may want to rename this tied-type since it is unlikely to be used elsewhere. I would also like the documentation to refer to this specifically for undamped notes. Notes that have an unattached tie due to repeat structure or being in an excerpt should use a pair of start and stop tied elements. I think that should also be added to the documentation here as it is not at all clear in the MusicXML 3.0 schemas.